uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/leave-status-id-model.js

```diff
@@ -0,0 +1,39 @@
+define([
+  'leave-absences/shared/modules/models',
+  'common/services/api/option-group',
+  'common/models/model'
+], function (models) {
+  'use strict';
+
+  models.factory('LeaveStatusID', [
```

Comment:
In Leave Request the option values can be resued multiple times, so instead of reinventing the wheel every time, I created a single service to take care of the same/